app/models/journeys/page_sequence.rb

```diff
@@ -116,12 +116,26 @@ def journey
     end
 
     def claim_submittable?
-      journey::ClaimSubmissionForm.new(
-        journey_session: journey::ClaimJourneySessionShim.new(
-          current_claim: @claim,
-          journey_session: @journey_session
-        )
-      ).valid?
+      journey::ClaimSubmissionForm.new(journey_session: shim).valid?
+    end
+
+    def student_loans_eligibility_checker
+      @eligibility_checker ||= Policies::StudentLoans::EligibilityChecker.new(shim.answers)
+    end
+
+    def shim
+      @shim ||= journey::ClaimJourneySessionShim.new(
+        current_claim: @claim,
+        journey_session: @journey_session
+      )
+    end
+
+    def eligibility_checkers
+      if journey == Journeys::TeacherStudentLoanReimbursement
+        [student_loans_eligibility_checker]
+      else
+        [claim]
+      end
```

Comment:
another one where we'll eventually want to do something like
```ruby
journey::POLICIES.map { |p| p::EligibilityChecker.new(answers) }
```

once we've finished the migration and don't require the shim